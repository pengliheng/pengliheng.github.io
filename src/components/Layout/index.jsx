import React, { Component } from 'react'
import './index.scss'

// import Header from '../Header';
import Side from '@/components/Side'
import Footer from '@/components/Footer'

export default class Layout extends Component {
	render() {
		const { children } = this.props
		return (
			<div className="Layout__container">
				{/* <Header /> */}
				<div className="Layout__content">
					<Side />
					{children}
				</div>
				<Footer />
			</div>
		)
	}
}
