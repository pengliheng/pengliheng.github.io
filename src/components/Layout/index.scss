.Layout__container {
  width: 100%;
  height: 100vh;
  display: flex;
  flex-direction: column;
  .Layout__content {
    margin: auto;
    flex: 1;
    // width: 100%;
    width: 1200px;
    display: flex;
    flex-direction: row;
  }
}