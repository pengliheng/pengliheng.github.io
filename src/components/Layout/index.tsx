import * as React from 'react'
import './index.scss'

// import Header from '../Header';
import Footer from '../../components/Footer/index'
import Side from '../../components/Side/index'

export default ({ children }: any) => (
  <div className="Layout__container">
    {/* <Header /> */}
    <div className="Layout__content">
      <Side />
      {children}
    </div>
    <Footer />
  </div>
)
