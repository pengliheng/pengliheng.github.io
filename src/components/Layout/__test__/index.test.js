// Link.react.test.js
import React from 'react'
import Layout from '../index.jsx'
import renderer from 'react-test-renderer'

it('renders correctly', () => {
	const tree = renderer.create(<Layout />).toJSON()
	expect(tree).toMatchSnapshot()
})
