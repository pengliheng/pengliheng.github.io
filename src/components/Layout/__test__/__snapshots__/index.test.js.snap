// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="Loading"
>
  <img
    alt="loading"
    src={Object {}}
  />
</div>
`;
